demonstrations/tutorial_fdhs.py

```diff
@@ -0,0 +1,500 @@
+    :align: center
+    :width: 70%
+    :target: javascript:void(0)
+
+Introduction
+------------
+
+The :doc:`KAK theorem </demos/tutorial_kak_theorem>` is an important result from Lie theory that states that any Lie group element :math:`U` can be decomposed
+as :math:`U = K_1 A K_2,` where :math:`K_{1, 2}` and :math:`A` are elements of two special sub-groups
+:math:`\mathcal{K}` and :math:`\mathcal{A},` respectively. In special cases, the decomposition simplifies to :math:`U = K A K^\dagger.`
+
+You can think of this KAK decomposition as a generalization of
+the singular value decomposition to Lie groups. For that, recall that the singular value decomposition states that any
+matrix :math:`M \in \mathbb{C}^{m \times n}` can be decomposed as :math:`M = U \Lambda V^\dagger,` where :math:`\Lambda`
+are the diagonal singular values and :math:`U \in \mathbb{C}^{m \times \mu}` and :math:`V^\dagger \in \mathbb{C}^{\mu \times n}`
+are left- and right-unitary with :math:`\mu = \min(m, n).`
+
+In the case of the KAK decomposition, :math:`\mathcal{A}` is an Abelian subgroup in which all its elements are commuting,
+just as is the case for diagonal matrices.
+
+We can use this general result from Lie theory as a powerful circuit decomposition technique.
+
+.. note:: We recommend a basic understanding of Lie algebras, see e.g. our :doc:`intro for quantum practitioners </demos/tutorial_liealgebra>`.
+    Otherwise this demo should be self-contained. For the mathematically inclined we further recommend our :doc:`demo on the KAK theorem </demos/tutorial_kak_theorem>`
+    that dives into the mathematical depths of the theorem and provides more background info.
+
+Goal
+----
+
+Unitary gates in quantum computing are described by the special unitary Lie group :math:`SU(2^n),` so we can use the KAK
+theorem to decompose quantum gates into :math:`U = K_1 A K_2.` While the mathematical statement is rather straightforward,
+actually finding this decomposition is not. We are going to follow the recipe prescribed in 
+`Fixed Depth Hamiltonian Simulation via Cartan Decomposition <https://arxiv.org/abs/2104.00728>`__ [#Kökcü]_, 
+which tackles this decomposition on the level of the associated Lie algebra via Cartan decomposition.
+
+In particular, we are going to consider the problem of time-evolving a Hermitian operator :math:`H` that generates the time-evolution unitary :math:`U = e^{-i t H}.`
+We are going to perform a special case of KAK decomposition, a "KhK decomposition" if you will, on the algebraic level in terms of
+
+.. math:: H = K^\dagger h_0 K.
+
+This then induces the KAK decomposition on the group level as
+
+.. math:: e^{-i t H} = K^\dagger e^{-i t h_0} K.
+
+Let us walk through an explicit example, doing theory and code side-by-side.
+
+For that, we are going to use the generators of the Heisenberg model Hamiltonian for :math:`n=4` qubits on a one-dimensional chain,
```

Comment:
@justinpickering there are good links for this on pennylane.ai

Reply:
The first one — this is a very recent PennyLane Demo and we would like to promote it more.

Reply:
I think this comes down to our priorities: do we want to prioritize our own content over external sites, even if the external link provides more descriptive context for readers? If we do, we can link to the PennyLane demo; if not, we link to the Wikipedia page. Given the context @Qottmann provided, I think the Wikipedia link is the most appropriate choice here.

```suggestion
For that, we are going to use the generators of the `Heisenberg model <https://en.wikipedia.org/wiki/Quantum_Heisenberg_model>`__ Hamiltonian for :math:`n=4` qubits on a one-dimensional chain,
```